ingest/src/test/java/com/connexta/ingest/IngestApplicationIntegrationTest.java

```diff
@@ -69,15 +77,33 @@ public void testSuccessfulTransformRequest() throws Exception {
                 .accept(MediaType.APPLICATION_JSON)
                 .contentType(MediaType.MULTIPART_FORM_DATA))
         .andExpect(status().isAccepted());
-
-    verify(restTemplate, times(1))
-        .postForObject(
-            eq("https://localhost/transform"), any(HttpEntity.class), eq(TransformResponse.class));
   }
 
-  // TODO Test what happens when an error occurs when hitting transform service
   @Test
-  public void testUnsuccessfulRequest() {}
+  public void testUnsuccessfulRequest() throws Exception {
+    server
+        .expect(requestTo("https://localhost/transform"))
+        .andRespond(
+            withStatus(HttpStatus.BAD_REQUEST)
```

Comment:
✏️ Should we have tests for each of the error codes that the Transform endpoint can return?
https://github.com/connexta/ion-transformation-api/blob/5f7ada923087ca6ce9334111bef59440129fd009/rest/specs/src/main/filtered-resources/openapi/openapi.yaml#L42-L56

Reply:
I was thinking about this, but the handler throws the same exception for all of them. It's fairly trivial to add in the tests for all of them, though, so I can add them. 👍

Reply:
I'd be fine saving writing those tests for when we implement contract testing. For now, could you at least write a code comment in the tests to prevent future confusion, like
```java
\\ The error handler throws the same exception for all non-202 status codes returned by the transformation endpoint
```
?

Reply:
Sure this works.